src/App.js

```diff
@@ -1,22 +1,39 @@
 import "./App.css";
-import { Switch, Route } from "react-router-dom";
+import { Route, NavLink, Switch } from "react-router-dom";
+import Profile from "./components/Profile";
+import Places from "./components/Places";
+import Map from "./components/Map";
 
 function App() {
   return (
     <div className="App">
-      <header class="header">Awesome Header</header>
-      <main>Awesome Content</main>
-      <Switch>
-        <Route path="/places"></Route>
-      </Switch>
-      <Switch>
-        <Route path="/profile"></Route>
-      </Switch>
-
-      <Switch>
-        <Route exact path="/"></Route>
-      </Switch>
-      <footer>Awesome Footer</footer>
+      <header className="header">
+        <Switch>
+          <Route path="/map">
+            <Map />
+          </Route>
+          <Route path="/places">
+            <Places />
+          </Route>
+          <Route path="/profile">
+            <Profile />
+          </Route>
+        </Switch>
```

Comment:
I think the `/` path is missing, so user get automatically redirected to the main page (it can be whatever is your main page -> either home or maybe even in map)

Reply:
the truth is that I used to have it before. I guess because I want my main page to be a plain one with no footer/header just a logo and a loading more button, I couldn't decide how to include it.